sunnyportal/requests.py

```diff
@@ -239,3 +239,26 @@ def __init__(self, token, oid, date, period, interval, total=False):
 
     def handle_response(self, data):
         return responses.EnergyBalanceResponse(data)
+
+
+class LogbookRequest(RequestBase):
+    def __init__(self, token, oid, date_from, info=True, warn=True, failure=True, error=True):
+        super().__init__(service="eventlist", token=token)
+        params = {
+            "culture": "en-gb",
+            "date-from": date_from.strftime("%Y-%m-%d"),
```

Comment:
Should date_from be made optional, assuming it will then return all logs?

Reply:
Confirmed that this works. I'll make it optional.